src/InputstreamDemux.cpp

```diff
@@ -386,8 +386,11 @@ void cVNSIDemux::StreamChange(cResponsePacket* resp)
       const char* language = resp->extract_String();
       uint32_t composition_id(resp->extract_U32());
       uint32_t ancillary_id(resp->extract_U32());
-      props.SetLanguage(language);
-      props.SetSubtitleInfo((composition_id & 0xffff) | ((ancillary_id & 0xffff) << 16));
+      if (strlen(language))
```

Comment:
Always qualify standard functions: `std::strlen`

Reply:
is the problem actually the string length or rather `language` being `nullptr`? If the latter, the better fix would be to change if `strlen(language)` to `if (language)`.

Reply:
The problem is indeed caused by an empty string